docs/docusaurus/docs/cloud/validations/manage_validations.md

```diff
@@ -72,6 +72,8 @@ To make the identification of Data Assets containing partitions easier, partitio
     
     - To view the run history of all Validations, select **All Runs** to view a graph showing the Validation run history for all columns.
 
+    - Note: items in run history with a calendar icon are run on a GX-managed schedule.
```

Comment:
I'd suggest putting this in an admonition box or a following paragraph, rather than leaving it as an item in the list.  All of the other items in this list are options for the user to choose from; this is not.

```suggestion
:::tip

Items in run history with a calendar icon are run on a GX-managed schedule.

:::
```
Alternative admonitions boxes are `note` or `info`.  You can add a label as well.